build.gradle

```diff
@@ -263,6 +263,10 @@ shadowJar {
     }
 }
 
+task localJar {
+    dependsOn shadowJar 
+}
+
 task sparkJar(type: ShadowJar) {
     configurations = [project.configurations.sparkConfiguration]
     classifier = 'spark'
```

Comment:
Do the symlinks still work?  I.e. gatk-protected.jar is a symlink to the local jar?